src/mqueryfront/src/status/StatusPage.js

```diff
@@ -22,8 +21,7 @@ class StatusPage extends Component {
     }
 
     componentDidMount() {
-        axios
-            .get(`${API_URL}/backend`)
+        api.get(`/backend`)
```

Comment:
```suggestion
        api.get("/backend")
```

(basing on previous changes)